packages/headless/dist/parsed_doc.json

```diff
@@ -0,0 +1,180 @@
+{
+  "controllers": [ "Hidden for this PR" ],
```

Comment:
Don't worry, the `parsed_doc.json` file is still ignored by git. I added it into this PR for the purpose of reviewing the JSON documentation generated by this PR, I will revert it after.